nuclear/message/include/message/format/math_types.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef MESSAGE_FORMAT_MATRIX_TYPES_HPP
```

Comment:
This should probably be in `utility` and included on-demand. This would make it more accessible to any part of the system (including other utilities) and doesn't add unnecessary symbols to the messages library. Also, given these are template types that have no instantiation inside of the messages library, there is a decent chance that these symbols wont even survive into the rest of the system